examples/dihedral/dihedral.yml

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env yason.py
```

Comment:
Add as unittest in `examples/CMakeLists.txt`; see how it's done for `bend`. Not sure we can run `compare.py` as a unittest as this would require `numpy` and other libraries installed.

Reply:
also remember to add `.yml` and `.out.json` to install target, again see bend example.

Reply:
Fixed with 8ff5d87.